test/onigumo_cli_test.exs

```diff
@@ -21,16 +21,20 @@ defmodule OnigumoCLITest do
       Onigumo.CLI.main(["Downloader"])
     end
 
-    test("run CLI with unknown argument") do
+    test("run CLI with invalid argument") do
```

Comment:
Maybe this is a separate change, but I see the change for consistency with the new “invalid switch” tests. It is okay to have it here.